include/superior_mysqlpp/types/datetime.hpp

```diff
@@ -87,15 +136,24 @@ namespace SuperiorMySqlpp
 
     namespace detail
     {
+        /**
+         * @brief Says that object Datetime can be used as a storage of type datetime as a variant for query parameter.
+         */
         template<>
         struct CanBindAsParam<BindingTypes::Datetime, Datetime> : std::true_type {};
 
+        /**
+         * @brief Says that object Datetime can be used as a storage of type datetime as a variant for query result.
+         */
         template<>
         struct CanBindAsResult<BindingTypes::Datetime, Datetime> : std::true_type {};
     }
 
 
 
+    /**
+     * Timestamp class derived from Datetime.
```

Comment:
Useless comment, please describe purpose of this class.

Reply:
ACK